app/main/plugins/core/cerebro/settings/Settings/components/input.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react'
+import styles from '../styles.css'
+import hotkeyStyles from '../Hotkey/styles.css'
```

Comment:
I think it makes sense to move hotkey into `components` directory and reuse styles for this component for all text inputs (so, we won't have separate css file for `Hotkey ` component)